e2e/keywords/volume.resource

```diff
@@ -9,11 +10,20 @@ Create a volume with ${size} GB and ${replica_count} replicas
     ${volume_name} =    create_volume   ${size}    ${replica_count}
     attach_volume    ${volume_name}
     Set Test Variable    ${volume_name}
+    Append To List    ${volume_list}    ${volume_name}
+
+Create volume ${idx} with size ${size} GB and ${replica_count} replicas
+    ${volume_name} =    create_volume   ${size}    ${replica_count}
```

Comment:
nit: `Create volume ${idx} with ${size} GB and ${replica_count} replicas`

Reply:
Refined in https://github.com/longhorn/longhorn-tests/pull/1529.